lib/registries/addon/branded/discover/styles.scss

```diff
@@ -0,0 +1,17 @@
+// Brand Variables
+// TODO:
+// - Find proper place to put these
+// - Figure out defaults
+:root {
+    --primary-color: #000;
+    --secondary-color: #666;
+    --navbar-logo-img-url: '';
+    --hero-logo-img-url: '';
+    --hero-background-img-url: '';
+}
```

Comment:
Where should I put these variables?

Reply:
hmm the concept of branding isn't strictly limited to registries, so i think these might make the most sense in `app/styles/_branding.scss`? (then imported in `app/styles/headers.scss`)

or the default could be handled by that modifier i suggested below -- so defaults aren't set on `:root` but instead on the application's wrapping div... i'm not sure, thoughts?

Reply:
That makes sense. Will there ever be a time when we don't want to render a branded registry? If there's a case like that then we can put a `branded` (or something like that) class somewhere near the application's wrapping div, and only add that class when we want branding.

Reply:
hmm I think ideally there's always an active brand -- most of the time it'll just be the default OSF brand, which matches the colors and such that are already there.